patches/external-kafka.sh

```diff
@@ -0,0 +1,250 @@
+      SENTRY_KAFKA_SSL_KEY_PATH: \${KAFKA_SSL_KEY_LOCATION:-}
+      SENTRY_KAFKA_SASL_MECHANISM: \${KAFKA_SASL_MECHANISM:-}
+      SENTRY_KAFKA_SASL_USERNAME: \${KAFKA_SASL_USERNAME:-}
+      SENTRY_KAFKA_SASL_PASSWORD: \${KAFKA_SASL_PASSWORD:-}
+    volumes:
+      - ./certificates/kafka:/kafka-certificates:ro
+  relay:
+    volumes:
+      - ./certificates/kafka:/kafka-certificates:ro
+  snuba-api:
+    <<: *snuba_defaults
+  snuba-errors-consumer:
+    <<: *snuba_defaults
+  snuba-outcomes-consumer:
+    <<: *snuba_defaults
+  snuba-outcomes-billing-consumer:
+    <<: *snuba_defaults
+  snuba-group-attributes-consumer:
+    <<: *snuba_defaults
+  snuba-replacer:
+    <<: *snuba_defaults
+  snuba-subscription-consumer-events:
+    <<: *snuba_defaults
+  snuba-transactions-consumer:
+    <<: *snuba_defaults
+  snuba-replays-consumer:
+    <<: *snuba_defaults
+  snuba-issue-occurrence-consumer:
+    <<: *snuba_defaults
+  snuba-metrics-consumer:
+    <<: *snuba_defaults
+  snuba-subscription-consumer-transactions:
+    <<: *snuba_defaults
+  snuba-subscription-consumer-metrics:
+    <<: *snuba_defaults
+  snuba-generic-metrics-distributions-consumer:
+    <<: *snuba_defaults
+  snuba-generic-metrics-sets-consumer:
+    <<: *snuba_defaults
+  snuba-generic-metrics-counters-consumer:
+    <<: *snuba_defaults
+  snuba-generic-metrics-gauges-consumer:
+    <<: *snuba_defaults
+  snuba-profiling-profiles-consumer:
+    <<: *snuba_defaults
+  snuba-profiling-functions-consumer:
+    <<: *snuba_defaults
+  snuba-spans-consumer:
+    <<: *snuba_defaults
+  web:
+    <<: *sentry_defaults
+  cron:
+    <<: *sentry_defaults
+  worker:
+    <<: *sentry_defaults
+  events-consumer:
+    <<: *sentry_defaults
+  attachments-consumer:
+    <<: *sentry_defaults
+  post-process-forwarder-errors:
+    <<: *sentry_defaults
+  subscription-consumer-events:
+    <<: *sentry_defaults
+  transactions-consumer:
+    <<: *sentry_defaults
+  metrics-consumer:
+    <<: *sentry_defaults
+  generic-metrics-consumer:
+    <<: *sentry_defaults
+  billing-metrics-consumer:
+    <<: *sentry_defaults
+  ingest-replay-recordings:
+    <<: *sentry_defaults
+  ingest-occurrences:
+    <<: *sentry_defaults
+  ingest-profiles:
+    <<: *sentry_defaults
+  ingest-monitors:
+    <<: *sentry_defaults
+  ingest-feedback-events:
+    <<: *sentry_defaults
+  monitors-clock-tick:
+    <<: *sentry_defaults
+  monitors-clock-tasks:
+    <<: *sentry_defaults
+  post-process-forwarder-transactions:
+    <<: *sentry_defaults
+  post-process-forwarder-issue-platform:
+    <<: *sentry_defaults
+  subscription-consumer-transactions:
+    <<: *sentry_defaults
+  subscription-consumer-metrics:
+    <<: *sentry_defaults
+  subscription-consumer-generic-metrics:
+    <<: *sentry_defaults
+EOF
+)
+if [[ -f "docker-compose.override.yml" ]]; then
+  echo "🚨 docker-compose.override.yml already exists. You will need to modify it manually:"
```

Comment:
Maybe try to be more helpful by sharing _what_ needs to be changed in the file? Like "you need to remove the `kafka` service" or something among those lines.

Reply:
Although I print the `docker-compose.override.yml` content, yeah, this might be helpful too.